plexpy/webserve.py

```diff
@@ -46,9 +46,13 @@ def serve_template(templatename, **kwargs):
 
     _hplookup = TemplateLookup(directories=[template_dir])
 
+    # Get the server name
+    pms_connect = pmsconnect.PmsConnect()
+    server_name = pms_connect.get_server_friendly_name()
```

Comment:
I don't particularly like this, it means we have to wait for a server response before we can load our home page. If the server is down or slow to respond every time you hit the page it's going to be slow.

I'd rather store this in the config file and refresh it on start up or with the scheduler in the background.